blocks/rich-text/index.js

```diff
@@ -386,24 +387,15 @@ export class RichText extends Component {
 	 *
 	 * Based on the selection of the text inside this element a position is
 	 * calculated where the toolbar should be. This can be used downstream to
-	 * absolutely position the toolbar. It does this by finding the closest
-	 * relative element.
+	 * absolutely position the toolbar.
 	 *
 	 * @param {DOMRect} position Caret range rectangle.
 	 *
 	 * @return {{top: number, left: number}} The desired position of the toolbar.
 	 */
 	getFocusPosition( position ) {
-		// Find the parent "relative" or "absolute" positioned container
-		const findRelativeParent = ( node ) => {
-			const style = window.getComputedStyle( node );
-			if ( style.position === 'relative' || style.position === 'absolute' ) {
-				return node;
-			}
-			return findRelativeParent( node.parentNode );
-		};
-		const container = findRelativeParent( this.editor.getBody() );
-		const containerPosition = container.getBoundingClientRect();
+		// The container is relatively positioned.
+		const containerPosition = this.containerRef.current.getBoundingClientRect();
```

Comment:
Did you try inline toolbars? I think those were the reason for using the `findRelativeParent` utility instead of hard-coding the parent.

Reply:
Just tried, and yes it works great 👍